code/modules/reagents/reagent_containers/cups/soda.dm

```diff
@@ -335,3 +335,11 @@
 		if("Moonlabor Malt's")
 			icon_state = "moonlabor"
 			desc = "Mars' underdog rice beer brand. Popular amongst the Yakuza, for reasons unknown."
+
+/obj/item/reagent_containers/cup/soda_cans/liquid_plasma
```

Comment:
Plasma should be mined not abused by vending machine tech to churn out especially since you can solidify it.

Reply:
just make a reagent that has a recipe of liquid and plasma and then put that reagent here, lets you cook with the drink effects too 😼